bookmarks.py

```diff
@@ -55,30 +55,31 @@ def connect_to_database(self) -> None:
             self.cursor = self.con.cursor()
         return None
 
-    def search(self, term) -> list:
+    def search(self, term = "") -> list:
         """Search for a term in the database and return the results."""
-        # VULN sql injection.
         query = (
             "SELECT A.title, B.url FROM moz_bookmarks AS A"
             + " JOIN moz_places AS B ON(A.fk = B.id)"
-            + ' WHERE A.title LIKE "%%%s%%"'
-        ) % term
+        )
+
+        args = term
```

Comment:
But what's the point of `args = term` if we don't need the args? It's just for avoiding the error.

When `term == ""` we don't need any rags so it should be an empty tuple.

It should be:
```python
args = ()
```
rather then:
```python
args = term
```

Reply:
that's a good point

Reply:
I pushed up a change I'll let you resolve this conversation if you're happy with the changes.